### ISSUE TITLE
No norming of descriptor vectors in loss function?

### ISSUE THREAD

**Issue by antopost** [eric-yyjau/pytorch-superpoint#84]
Hi, I was looking at the [descritpor_loss](https://github.com/eric-yyjau/pytorch-superpoint/blob/5eb75d74df27c07f6e7311df8f167e2a9c01a798/utils/utils.py#L739) function and I could not find where the descriptor vectors are normed before the [dot multiplication](https://github.com/eric-yyjau/pytorch-superpoint/blob/5eb75d74df27c07f6e7311df8f167e2a9c01a798/utils/utils.py#L815).

Edit: Didn't realize that the descriptor head outputs already normed descriptor vectors, so the loss function makes sense as it is
